curvefs/proto/common.proto

```diff
@@ -47,4 +47,5 @@ message PartitionInfo {
     required uint32 partitionId = 4;
     required uint64 start = 5;
     required uint64 end = 6;
```

Comment:
add annotation to explain `end` whether inclusive or exclusive

Reply:
add comment     "// partion manage inodeid range [start, end]"